configs/acra-addzone.yaml

```diff
@@ -86,15 +86,48 @@ vault_connection_api_string:
 # KV Secret Path (secret/) for reading ACRA_MASTER_KEY from HashiCorp Vault
 vault_secrets_path: secret/
 
-# Path to CA certificate for HashiCorp Vault certificate validation
-vault_tls_ca_path: 
```

Comment:
unfortunately, it is breaking change and we should leave it. and probably deal with it as we do for acra-server, with common parameter used until specified for explicit side (client/database)

Reply:
Great notion, agree!